### PR TITLE
i18n(id): Update `environmental-impact.md` translation

### PR DESCRIPTION
**Pull request by iyyafi**
#### Description

- Update translation for `environmental-impact.md` page in Bahasa Indonesia

### CODE REVIEW

File: docs/src/content/docs/id/environmental-impact.md [hunk 1]
```diff
@@ -84,20 +84,20 @@ Memilih penyedia hosting yang menggunakan energi terbarukan berarti emisi karbon
...
+| [Sphinx][sx-carbon]         | 0.06g                     |     A+    |
+| [VitePress][vp-carbon]      | 0.07g                     |     A+    |
+| [Docus][dc-carbon]          | 0.09g                     |     A+    |
+| [docsify][dy-carbon]        | 0.10g                     |     A     |
+| [Nextra][nx-carbon]         | 0.11g                     |     A     |
+| [MkDocs][mk-carbon]         | 0.19g                     |     B     |
+| [Docusaurus][ds-carbon]     | 0.21g                     |     B     |
+| [GitBook][gb-carbon]        | 0.43g                     |     C     |
+| [Mintlify][mt-carbon]       | 1.22g                     |     F     |
+<small>Data dikumpulkan pada 22 July 2024. Klik link untuk melihat angka terkini.</small>
```
Comment (HiDeoo) [inline docs/src/content/docs/id/environmental-impact.md:100]:
```suggestion

<small>Data dikumpulkan pada 22 July 2024. Klik link untuk melihat angka terkini.</small>
```

To match the original version formatting

Comment (HiDeoo) [inline docs/src/content/docs/id/environmental-impact.md:100]: Just to double check, is "July" expected here or something that could have been missed from the English version?

File: docs/src/content/docs/id/environmental-impact.md [hunk 2]
```diff
@@ -84,20 +84,20 @@ Memilih penyedia hosting yang menggunakan energi terbarukan berarti emisi karbon
...
 
 [sl-carbon]: https://www.websitecarbon.com/website/starlight-astro-build-getting-started/
 [vp-carbon]: https://www.websitecarbon.com/website/vitepress-dev-guide-what-is-vitepress/
```
Comment (HiDeoo) [inline docs/src/content/docs/id/environmental-impact.md:103]: Looks like the [`mt-carbon` reference](https://github.com/withastro/starlight/pull/2136/files#diff-f2d1d6c7f449448ddeca5c7e9315cc6d0066c2d2cf728cdfb0b3c289c2550bb4R118) is missing from this translation to make it a valid link.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few comments and questions to fix a few minor issues but otherwise, the update looks good to me.

File: docs/src/content/docs/id/environmental-impact.md (hunk 1 above)
Comment (iyyafi) [reply docs/src/content/docs/id/environmental-impact.md:100]: Ah, i see. I miss this one. Thanks, will fix soon.

File: docs/src/content/docs/id/environmental-impact.md (hunk 2 above)
Comment (iyyafi) [reply docs/src/content/docs/id/environmental-impact.md:103]: I don't see this before, thank you for tell me this.

**Review by iyyafi — COMMENTED** [review]
Hi, thank you for the review. I have push some updates, hope it will enough get approve.

File: docs/src/content/docs/id/environmental-impact.md [hunk 3]
```diff
@@ -84,20 +84,20 @@ Memilih penyedia hosting yang menggunakan energi terbarukan berarti emisi karbon
...
+| [Sphinx][sx-carbon]         | 0.06g                     |     A+    |
+| [VitePress][vp-carbon]      | 0.07g                     |     A+    |
+| [Docus][dc-carbon]          | 0.09g                     |     A+    |
+| [docsify][dy-carbon]        | 0.10g                     |     A     |
+| [Nextra][nx-carbon]         | 0.11g                     |     A     |
+| [MkDocs][mk-carbon]         | 0.19g                     |     B     |
+| [Docusaurus][ds-carbon]     | 0.21g                     |     B     |
+| [GitBook][gb-carbon]        | 0.43g                     |     C     |
+| [Mintlify][mt-carbon]       | 1.22g                     |     F     |
+<small>Data dikumpulkan pada 22 Juli 2024. Klik link untuk melihat angka terkini.</small>
```
Comment (HiDeoo) [inline docs/src/content/docs/id/environmental-impact.md:100]:
```suggestion
| [Mintlify][mt-carbon]       | 1.22g                     |     F     |

<small>Data dikumpulkan pada 22 Juli 2024. Klik link untuk melihat angka terkini.</small>
```

**Review by HiDeoo — APPROVED** [review]
Thanks for updating the branch. I took the liberty of fixing the table formatting so that it displays correctly but everything else looks great to me. Thanks again for your contribution :raised_hands:

**Pull request merged**